Lib/difflib.py

```diff
@@ -365,6 +365,12 @@ def find_longest_match(self, alo=0, ahi=None, blo=0, bhi=None):
             ahi = len(a)
         if bhi is None:
             bhi = len(b)
+
+        # Check if both sequences are the same before executing rest of the
+        # method.
+        if a[alo:ahi] == b[blo:bhi]:
```

Comment:
Is this covered in tests? if not, please add a test, otherwise no need to add a large one.